pyproject.toml

```
@@ -122,6 +122,46 @@ requires = ["poetry-core>=1.1.0"]
 build-backend = "poetry.core.masonry.api"
 
 
+[tool.ruff]
+fix = true
```

```suggestion
fix = true
unfixable = [
    "ERA",  # do not autoremove commented out code
]
```

I don't like it when ruff removes commented out code. During development I often comment out code and sometimes run pre-commit.

---

Didn't notice the `unfixable` option before. Added.

---

Didn't notice the `unfixable` option before. Added.